data/json/items/comestibles/dairy.json

```diff
@@ -541,5 +541,22 @@
     "flags": [ "EDIBLE_FROZEN" ],
     "fun": 2,
     "vitamins": [ [ "calcium", 19 ], [ "iron", 5 ], [ "milk_allergen", 1 ] ]
+  },
+  {
+    "id": "rennet",
+    "type": "COMESTIBLE",
+    "comestible_type": "INVALID",
+    "category": "chems",
+    "name": { "str_sp": "rennet" },
+    "description": "Rennet.",
+    "symbol": "~",
+    "color": "white",
+    "volume": "250 ml",
+    "weight": "250 g",
```

Comment:
[This product safety sheet](https://tecnolatte.it/uploads/prodotti/schede/23_2.PDF) describes the density of liquid rennet as between 1.1 and 1.15 kg/L, "[i]n relation to the NaCl content in solution". I'm not sure what that means exactly, but this is the closest I've gotten to the most-appropriate value. [This listing for the same product](https://tecnolatte.it/eng/e-commerce/product/681/rennet-natural-liquid-calf-graco-abr-bottle-500-gr) should describe whether that's the right item to base it on.

(Related research: [this paper](https://www.sciencedirect.com/science/article/pii/S0022030220300758) describes rennet *casein*, which I believe to be the resulting *cheese*, not the rennet used. [This product page](https://www.wardsci.com/store/product/8885799/rennet-tablets) lists the density of rennet *tablets* as 1.3 g/mL.)

Reply:
Rennet, as a protein, is 'a little heavier than water', so the range of 1.1 kg/L sounds closest, and most liquid rennet is mixed with enough water that the density of water is more paramount than the density of the enzyme.

"NaCl" is just 'how salty the water is'. Sodium chloride. Our rennet recipe is relatively salty, since we aren't using any other preservatives.

Caesin is just one of the proteins in milk, it's separated as part of curdling! It is broadly the reason that milk and cheese are considered nutritious and butter isn't as nutritious.